dashboard/app/helpers/users_helper.rb

```diff
@@ -92,7 +92,8 @@ def user_summary(user)
   end
 
   # Merge the progress for the specified script and user into the user_data result hash.
-  private def merge_script_progress(user_data, user, script, exclude_level_progress = false)
+  private def get_script_progress(user, script, exclude_level_progress = false)
```

Comment:
nit: update comment (no longer merge), and ideally add YARD style `@param` and `@return` descriptors